Makefile

```diff
@@ -42,20 +42,7 @@ BUF_VERSION_URL := https://raw.githubusercontent.com/vdaas/vald/main/versions/BU
 SHADOW_ROOT       = vald
 SHADOW_PROTO_ROOT = $(SHADOW_ROOT)/$(SHADOW_ROOT)
 
-PROTOS = \
-	v1/agent/core/agent.proto \
-	v1/filter/egress/egress_filter.proto \
-	v1/filter/ingress/ingress_filter.proto \
-	v1/vald/filter.proto \
-	v1/vald/insert.proto \
-	v1/vald/object.proto \
-	v1/vald/index.proto \
-	v1/vald/remove.proto \
-	v1/vald/search.proto \
-	v1/vald/update.proto \
-	v1/vald/upsert.proto \
-	v1/payload/payload.proto
-PROTOS := $(PROTOS:%=$(PROTO_ROOT)/%)
+PROTOS = $(shell find $(PROTO_ROOT)/v1 -name '*.proto')
```

Comment:
```suggestion
PROTOS = $(shell find $(PROTO_ROOT)/v1 -name '*.proto'  \( -path '*/vald/*' -o -path '*/agent/*' \))
```

Reply:
Can you ignore discoverer & meta & mirror & sidecar?

Reply:
Thank you for the feedback.I added the ignore list to convert it into find command options. 
This should ensure that the specified components are excluded from being generated.

Reply:
I think it would be ideal to manage only the targets to build (vald, agent, rpc) instead of managing the　Ignore list (discoverer, meta, mirror, sidecar, etc...)

Reply:
I changed it to be managed by a target list. Does this look correct?
https://github.com/vdaas/vald-client-python/pull/41/commits/a93fb213ed75614679f3f30727cd734dda0af28e